zbus-lockstep/src/lib.rs

```diff
@@ -123,7 +123,7 @@ pub fn get_signal_body_type<'a>(
                 .collect::<String>()
         }
     };
-    Ok(Signature::from_string_unchecked(signature))
+    Ok(Signature::from_str(&signature).map_err(|_| "Invalid signature")?)
```

Comment:
Is "Invalid signature" (for an error) clearer than the error supplied by the `FromStr` implementation?

Reply:
If not, maybe we could just '?' our way out.